erpnext/patches/v17/add_purchase_order_custom_fields_002.py

```
@@ -0,0 +1,50 @@
+                "description": "Special instructions for procurement"
+            }
+        ]
+    }
+    
+    try:
+        create_custom_fields(custom_fields, update=True)
+        frappe.db.commit()
+        print("Custom fields added to Purchase Order")
+        
+    except Exception as e:
+        print(f"Error: {str(e)}")
```

Ruff (0.15.1)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)